app/uk/gov/hmrc/helptosave/repo/EmailStore.scala

```diff
@@ -132,13 +136,12 @@ class MongoEmailStore @Inject() (mongo:   ReactiveMongoComponent,
 
   }
 
-  private[repo] def doUpdate(encryptedEmail: String, nino: NINO)(implicit ec: ExecutionContext): Future[Option[EmailData]] =
-    collection.findAndUpdate(
-      BSONDocument("nino" -> nino),
-      BSONDocument("$set" -> BSONDocument("email" -> encryptedEmail)),
-      fetchNewObject = true,
-      upsert         = true
-    ).map(_.result[EmailData])
+  private[repo] def doUpdate(encryptedEmail: String, nino: NINO)(implicit ec: ExecutionContext): Future[Boolean] =
```

Comment:
would we need to delete by regex when deleting emails as well?

Reply:
Yes as its possible it won't be picked up if a different suffix is used